### PR TITLE
Fix hls player

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Don't cache video el, as it gets recreated

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by balloob — APPROVED** [review]
Can we add a comment why it exists?

**Review by zsarnett — APPROVED** [review]

**Review by nick2525 — COMMENTED** [review]
I didn't see, where you transfer changes. I see you deleted changes, but I don't know Typescript //Oh, now I understood, you changed video cache flag

**Review by nick2525 — APPROVED** [review]

File: src/components/ha-hls-player.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ class HaHLSPlayer extends LitElement {
   @property({ type: Boolean, attribute: "allow-exoplayer" })
   public allowExoPlayer = false;
 
-  @query("video", true) private _videoEl!: HTMLVideoElement;
```
Comment (balloob) [inline src/components/ha-hls-player.ts:41]: Please add a comment

**Comment by abmantis** [comment]
If this fixes home-assistant/android#979 maybe it should be included on the current version as a point release? It is draining battery and eating a lot of traffic.

**Pull request merged**